tests/storage/export.go

```diff
@@ -197,6 +198,10 @@ var _ = SIGDescribe("Export", func() {
 				},
 			},
 		})
+		if pod.Spec.SecurityContext == nil {
+			pod.Spec.SecurityContext = &k8sv1.PodSecurityContext{}
+		}
```

Comment:
missing newline after `if`

Reply:
Sorry, when I will have more free bw, we should add a linter for that

Reply:
hmm it's actually intended, didn't know we enforce these?

Reply:
I see a mixed style in our code base and its something I would like to avoid. I am used to give a breath space for the if conditions blocks. I will try to create/enable a linting for that

Reply:
Actually it doesn't have to be logically tied because you assign the `FSGroup` also in case the `SecurityContext` isn't `nil` however I won't insist on that.